src/frontend/utils/mappingTilKontrakt/barnV7.ts

```diff
@@ -0,0 +1,153 @@
+            case undefined:
+                return ERegistrertBostedType.IKKE_FYLT_INN;
+            case true:
+                return ERegistrertBostedType.REGISTRERT_SOKERS_ADRESSE;
+            case false:
+                return ERegistrertBostedType.REGISTRERT_ANNEN_ADRESSE;
+            default:
+                return ERegistrertBostedType.IKKE_FYLT_INN;
+        }
+    };
+
+    return {
+        navn: søknadsfeltBarn(
+            intl,
+            'pdf.barn.navn.label',
+            sammeVerdiAlleSpråk(navn ?? `Barn ${formaterFnr(ident)}`),
+            barn
+        ),
+        ident: søknadsfeltBarn(
+            intl,
+
+            'pdf.barn.ident.label',
+            ident ? sammeVerdiAlleSpråk(ident) : hentTekster('pdf.barn.ikke-oppgitt'),
+            barn
+        ),
+        registrertBostedType: søknadsfeltBarn(
+            intl,
+
+            'hvilkebarn.barn.bosted',
+            sammeVerdiAlleSpråk(registertBostedVerdi()),
+            barn
+        ),
+        alder: søknadsfeltBarn(
+            intl,
+
+            'pdf.barn.alder.label',
+            alder
+                ? hentTekster('felles.år', { alder })
+                : sammeVerdiAlleSpråk(AlternativtSvarForInput.UKJENT),
+            barn
+        ),
+        utenlandsperioder: utenlandsperioder.map((periode, index) =>
+            utenlandsperiodeTilISøknadsfelt(intl, periode, index + 1, barn)
+        ),
+
+        eøsBarnetrygdsperioder: barn.eøsBarnetrygdsperioder.map((periode, index) =>
+            tilIEøsBarnetrygsperiodeIKontraktFormat({
+                intl,
+                periode,
+                periodeNummer: index + 1,
+                tilbakeITid: periode.mottarEøsBarnetrygdNå.svar === ESvar.NEI,
+                barn,
+            })
+        ),
+
+        andreForelder: andreForelder
+            ? andreForelderTilISøknadsfeltV7(intl, andreForelder, barn)
+            : null,
+        spørsmål: {
+            ...spørmålISøknadsFormat(typetBarnSpørsmål, {
+                navn: barnetsNavnValue(barn, intl),
+                barn: barnetsNavnValue(barn, intl),
+            }),
+            [barnDataKeySpørsmål.søkerForTidsromSluttdato]: søknadsfeltBarn(
+                intl,
+
+                språktekstIdFraSpørsmålId(OmBarnetSpørsmålsId.søkerForTidsromSluttdato),
+                sammeVerdiAlleSpråkEllerUkjentSpråktekst(
+                    søkerForTidsromSluttdato.svar,
+                    omBarnetSpørsmålSpråkId[OmBarnetSpørsmålsId.søkerForTidsromSluttdatoVetIkke]
+                ),
+                barn
+            ),
+
+            [barnDataKeySpørsmål.institusjonOppholdSluttdato]: søknadsfeltBarn(
+                intl,
+
+                språktekstIdFraSpørsmålId(OmBarnetSpørsmålsId.institusjonOppholdSluttdato),
+                sammeVerdiAlleSpråkEllerUkjentSpråktekst(
+                    institusjonOppholdSluttdato.svar,
+                    omBarnetSpørsmålSpråkId['institusjon-opphold-ukjent-sluttdato']
```

Comment:
Antar at dette bare er kopiert men har vi ikke en tekst for vet ikke i OmBarnetSpørsmålsId?